website/pages/index.tsx

```diff
@@ -1,5 +1,327 @@
+  padding: unset;
+  cursor: pointer;
+  position: relative;
+  color: ${uiColors.gray.dark3};
+`;
+
+const sharedHoverInteraction = css`
+  &:hover {
+    background-color: ${uiColors.gray.light3};
+    border: 1px solid ${uiColors.gray.light1};
+    box-shadow: 0 0 0 0px ${uiColors.gray.light1},
+      2px 16px 20px -10px rgba(0, 0, 0, 0.2);
+    transform: scale(1.05);
+    z-index: 99999;
+  }
+`;
+
+const previewWrapper = css`
+  ${container}
+  overflow: hidden;
+  transition: transform 300ms ease-in-out;
+  ${sharedHoverInteraction}
+
+  &:hover {
+    & > div {
+      opacity: 1;
+    }
+  }
+`;
+
+const overlineContainer = css`
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  padding-left: ${spacing[3]}px;
+  padding-bottom: ${spacing[3]}px;
+  transition: opacity 300ms ease-in-out;
+
+  ${mq({
+    opacity: [1, 1, 0, 0],
+  })}
+`;
+
+const overlineColor = css`
+  color: ${uiColors.gray.dark1};
+`;
+
+const marketingWrapper = css`
+  ${boxShadow}
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  position: relative;
+  transition: transform 300ms ease-in-out;
+  ${sharedHoverInteraction}
+`;
+
+const textWrapper = css`
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  font-weight: medium;
+  padding-top: ${spacing[4]}px;
+  padding-left: ${spacing[4]}px;
+  text-align: left;
+  overflow: hidden;
+
+  ${mq({
+    fontSize: ['24px', '60px', '60px', '60px'],
+  })}
+`;
+
+const lgHeight = css`
+  ${mq({
+    height: ['calc(100vw / 2)', '350px', '350px', '350px'],
+  })}
+`;
+
+const smHeight = css`
+  ${mq({
+    height: ['calc(100vw / 2)', '175px', '175px', '175px'],
+  })}
+`;
+
+const halfWidth = css`
+  width: 50%;
+`;
+
+const secondRowContainer = css`
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+interface ComponentPreviewProps {
+  path: string;
+  background: string;
+  content?: string;
+  children?: string;
+  className?: string;
+}
+
+function ComponentPreview({
+  path,
+  background,
+  content,
+  className,
+}: ComponentPreviewProps) {
+  const { push } = useRouter();
+  return (
+    <div className={className}>
+      <button className={previewWrapper} onClick={() => push(path)}>
+        <img
+          src={background}
+          alt={`Learn more about ${content} component`}
+          width="100%"
+        />
+        <div className={overlineContainer}>
+          <Overline className={overlineColor}>{content}</Overline>
+        </div>
+      </button>
+    </div>
+  );
+}
+
+interface MarketingPreview {
+  href: string;
+  children: string;
+  background: string;
+}
+
+function MarketingPreview({ href, children, background }: MarketingPreview) {
+  return (
+    <div className={lgHeight}>
+      <a
+        href={href}
+        className={container}
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <div className={marketingWrapper}>
+          <img
+            src={background}
+            alt=""
+            aria-hidden="true"
+            height="100%"
```

Comment:
why not css?

Reply:
If you feel strongly I can change this

Reply:
It would make updating and debugging styles easier if they're consistently defined via CSS

Reply:
👍